### ISSUE TITLE
SIC Code

### ISSUE THREAD

**Issue by tscottjendev** [Jersey-Electricity-plc/jwissues#19]
Should this be part of the base or would it be better as an add-on app?

If it is not used by everyone then having it everywhere can be confusing.

**Comment by RPCJendev** [comment]
I would agree that not all clients would want it and you could also argue that the term SIC code standing for Standard Industrial Classification is called something different according to country? Generally would be used for some sort of analysis.
